protocol/triple/triple_invoker.go

```diff
@@ -136,36 +143,50 @@ func (ti *TripleInvoker) Invoke(ctx context.Context, invocation protocol.Invocat
 	return &result
 }
 
+func mergeAttachmentToOutgoing(ctx context.Context, attachments map[string]interface{}) (context.Context, error) {
```

Comment:
Since you extract attachments merge logic from ```parseInvocation```. Why do not extract entire logics about processing attachments?

Reply:
i think u are right.